src/data/Vault.vala

```diff
@@ -5,6 +5,7 @@ namespace Cryptor.Data {
         public string mount_point { get; set; }
         public string mode { get; set; }
         public bool reverse { get; set; }
+        public string custom_options { get; set; }
```

Comment:
We should give this a default value of an empty string.
It's currently crashing for me on mount (if not present in the json config)

```suggestion
        public string custom_options { get; set; default = ""; }
```

With that also the "two null checks" can be removed.